common/thorhelper/csvsplitter.cpp

```diff
@@ -181,11 +191,8 @@ void CSVSplitter::init(unsigned _maxColumns, size32_t _maxCsvSize, const char *q
     if (escapes)
         addActionList(matcher, escapes, ESCAPE);
 
-    if (preserveWhitespace)
```

Comment:
You have inverted the test compared to the original code, I think - was the original wrong?